docs/Documentation/Environment/lmod.md

```diff
@@ -0,0 +1,149 @@
+# Environment Modules on NREL HPC
+
+Kestrel and Eagle uses the *Lmod* environment modules system to easily manage software environments. 
+Modules facilitate the use of different versions of applications, libraries, and toolchains, which enables support of multiple package versions concurrently.
+
+Modules typically just set environment variables that one might traditionally do manually by, for example, adding `export` or `setenv` commands to their login script. 
+Modules add the ability to back out changes in an orderly manner as well, so users can change their environment in a reversible way.
+
+Modulefiles can be hosted in any location that the user makes known to the Lmod system via the module use command. 
+By copying the reference modules, we set up into custom locations and modifying to your own tastes, you can host personal, project, or multi-project collections that you have complete control over.
+
+Our production location for module examples is `/nopt/nrel/apps/modules/default` on Kestrel and `/nopt/nrel/apps/modules/default` on Eagle. 
```

Comment:
Instead of listing the paths  is there a way to explain to users how to see the production location on whichever system they are on?